Windows Community Toolkit.sln

```diff
@@ -19,8 +19,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "UI", "UI", "{F1AFFFA7-28FE-
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{B30036C4-D514-4E5B-A323-587A061772CE}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Toolkit.Uwp.Services", "Microsoft.Toolkit.Uwp.Services\Microsoft.Toolkit.Uwp.Services.csproj", "{7189A42D-6F1A-4FA3-8E00-E2C14FDF167A}"
```

Comment:
Why the removal?

Reply:
Twitter implementation moved from Microsoft.Toolkit.Uwp.Services to Microsoft.Toolkit.Services.

Reply:
Looks like the project was removed and then added again. That's not necessary, please restore this file as it was - not relevant to this PR.